library/cpp/sanitizer/include/CMakeLists.linux-x86_64.txt

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
let's remove all the cmake files under `library/cpp/sanitizer` from the main branch